packages/features/ee/workflows/api/scheduleSMSReminders.ts

```diff
@@ -48,7 +48,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
   const dateInSevenDays = dayjs().add(7, "day");
 
-  unscheduledReminders.forEach(async (reminder) => {
+  for (const reminder of unscheduledReminders) {
```

Comment:
I'm assuming we don't care about these updates being run out of order right? If that's so then LGTM. We can either:
- [Run them in serial](https://decembersoft.com/posts/promises-in-serial-with-array-reduce/) or...
- Run them in parallel with Promise.all